generate-release-notes.js

```diff
@@ -39,6 +40,10 @@ async function generateReleaseNotes({
 
 	const commitEntries = [];
 	if (commits.length === 0) {
+		if (skipOnEmpty) {
+			return null;
```

Comment:
```suggestion
			return;
```

Reply:
If `return` is used without `null` then the return value will be `undefined` and will need to be changed in `index.js` as well.

`undefined` typically represents a value not having been explicitly defined, but since this `null` value is explicit in order to compare against in the index.js and differentiate against string values, I think it makes more sense to keep it as is.

Let me know your thoughts on this. I'm happy to change it if you still say so.

Reply:
> Let me know your thoughts on this. I'm happy to change it if you still say so.

Yes please

https://github.com/sindresorhus/meta/discussions/7